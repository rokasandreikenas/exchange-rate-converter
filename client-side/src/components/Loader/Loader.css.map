{
    "version": 3,
    "mappings": "AAAA,AAAA,OAAO,CAAC;EACJ,OAAO,EAAE,IAAI;EACb,WAAW,EAAE,IAAI;EACjB,YAAY,EAAE,IAAI;EAClB,WAAW,EAAE,IAAI;EACjB,MAAM,EAAE,KAAK;EACb,KAAK,EAAE,KAAK;CACf",
    "sources": [
        "Loader.scss"
    ],
    "names": [],
    "file": "Loader.css"
}