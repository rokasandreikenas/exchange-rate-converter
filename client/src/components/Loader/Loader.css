.loader {
  display: -webkit-box;
  display: -ms-flexbox;
  display: flex;
  margin-left: auto;
  margin-right: auto;
  padding-top: 50px;
  height: 150px;
  width: 150px;
}
/*# sourceMappingURL=Loader.css.map */