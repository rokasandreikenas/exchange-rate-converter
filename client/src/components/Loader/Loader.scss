.loader {
    display: flex;
    margin-left: auto;
    margin-right: auto;
    padding-top: 50px;
    height: 150px;
    width: 150px;
}
