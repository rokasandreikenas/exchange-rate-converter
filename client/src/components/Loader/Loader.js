import React from "react";
import IBMLogo from "../../assets/icons8-ibm.svg";
import "./Loader.scss";

const Loader = () => {
  return <img src={IBMLogo} alt="IBM logo" className="loader" />;
};

export default Loader;
