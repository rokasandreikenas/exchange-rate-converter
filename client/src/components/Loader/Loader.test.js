import React from "react";
import Loader from "./Loader";
import { shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";

describe("Loader", () => {
  it("renders Loader component correctly ", () => {
    const output = shallow(<Loader />);
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});
