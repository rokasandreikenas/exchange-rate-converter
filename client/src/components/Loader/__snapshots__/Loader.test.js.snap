// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader renders Loader component correctly  1`] = `
<img
  alt="IBM logo"
  className="loader"
  src="icons8-ibm.svg"
/>
`;
